<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt;
font-weight: bold;
color: rgb(213, 0, 0);</string>
   </property>
   <property name="text">
    <string>GIỎ HÀNG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>641</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>461</width>
     <height>201</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #ffffff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="list_widget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>501</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #ffffff;</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt;
color: rgb(216, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Thanh toán:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thanhToan">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>200</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background: rgb(201, 0, 0);
border-radius: 10px;
font-size: 20pt;</string>
   </property>
   <property name="text">
    <string>Thanh toán</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tongSoTien">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>560</y>
     <width>161</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt;
color: rgb(216, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>0 VNĐ</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>list_widget</zorder>
  <zorder>text</zorder>
  <zorder>btn_thanhToan</zorder>
  <zorder>lb_tongSoTien</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
